src/main.tsx

```diff
@@ -0,0 +1,50 @@
+// main.tsx
+
+import { StrictMode } from "react";
+import { createRoot } from "react-dom/client";
+import { RouterProvider, createHashRouter } from "react-router-dom";
+
+import "./index.css";
+
+import TimersView from "./views/TimersView";
+import DocumentationView from "./views/DocumentationView";
+import AddTimer from "./views/AddTimer";
+import PageIndex from "./components/pageIndex/PageIndex";
+import ErrorBoundary from "./components/errorBoundary/ErrorBoundary";
+import { TimerProvider } from "./context/TimerContext";
+
+const router = createHashRouter(
+    [
+        {
+            path: "/",
+            element: <PageIndex />,
+            errorElement: <div>An error occurred while loading this page.</div>,
+            children: [
+                { index: true, element: <TimersView /> },
+                { path: "/docs", element: <DocumentationView /> },
```

Comment:
child routes do not need the initial forward slash, they are by default nested under the root path